<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MyBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditID">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditYear"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Пол</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboSex">
        <item>
         <property name="text">
          <string>Man</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyBrowser</class>
   <extends>QTextBrowser</extends>
   <header location="global">mybrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SearchEmploye()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddEmploye()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SearchEmploye()</slot>
  <slot>AddEmploye()</slot>
 </slots>
</ui>
